environments/nanopore/environment.yml

```diff
@@ -16,11 +16,7 @@ dependencies:
   - fastqc=0.11.9
   - multiqc=1.11
   - muscle=3.8
+  - pycoqc
   - pip:
       - pandas == 1.0.1
       - scikit-learn >= 0.23.1
-      - git+https://github.com/cov-lineages/pangolin.git
-      - git+https://github.com/cov-lineages/constellations.git
-      - git+https://github.com/cov-lineages/scorpio.git
-      - git+https://github.com/cov-lineages/pangolin-data.git
-  - muscle=3.8
```

Comment:
Are you sure this line can be removed? We had a very strange error that we got another version of muscle (even though muscle=3.8 was stated above). The only way to fix it was adding muscle one more time at 3.8 the last thing that is done.

Reply:
Yes, muscle=3.8 is already at line18, why I need to add it twice!

Reply:
it works for me, otherwise we can just add this on last.

Reply:
> Yes, muscle=3.8 is already at line18, why I need to add it twice!

We had errors caused by muscle 3.8 was not running, even though muscle=3.8 was clearly added at line 18. We struggled with the error a lot and the only way to fix it was adding the last line.